optaweb-employee-rostering-backend/src/test/java/org/optaweb/employeerostering/solver/AbstractSolverTest.java

```diff
@@ -119,11 +117,11 @@ public void testFeasibleSolution() {
         Roster roster = rosterGenerator.generateRoster(10, 7);
 
         roster = solver.solve(roster);
-        assertNotNull(roster.getScore());
+        assertThat(roster.getScore()).isNotNull();
         // Due to overconstrained planning, the score is always feasible
-        assertTrue(roster.getScore().isFeasible());
-        assertFalse(roster.getShiftList().isEmpty());
-        assertTrue(roster.getShiftList().stream().anyMatch(s -> s.getEmployee() != null));
+        assertThat(roster.getScore().isFeasible()).isTrue();
+        assertThat(roster.getShiftList()).isNotEmpty();
+        assertThat(roster.getShiftList()).anyMatch(s -> s.getEmployee() != null);
```

Comment:
You may consider chaining these like so:

    assertThat(roster.getShiftList())
        .isNotEmpty()
        .anyMatch(s -> s.getEmployee() != null);

Also, you may want to look into `SoftAssertions` if you're checking multiple independent assertions in the same test.

Reply:
I will probably do it next sprint; a lot of the tests will be refactor to unit tests to reduce dependency on spring boot and decrease the time it takes for tests